src/QsCompiler/Tests.LanguageServer/TestUtils.cs

```diff
@@ -107,16 +107,13 @@ internal static int GetRangeLength(VisualStudio.LanguageServer.Protocol.Range ra
 
         internal static void ApplyEdit(TextDocumentContentChangeEvent change, ref List<string> content)
         {
-            if (content == null)
-            {
-                throw new ArgumentNullException(nameof(content));
-            }
             if (!content.Any())
             {
                 throw new ArgumentException("the given content has to have at least on line");
             }
 
-            Assert.IsTrue(IsValidRange(change?.Range) && change.Text != null);
+            Assert.IsNotNull(change);
```

Comment:
I think it is not possible for `change` to be null here because it is not a nullable `List<string>`, right? If that's the case, this Assert seems unnecessary.